assets/sass/patterns/organisms/site-header.scss

```diff
@@ -76,6 +76,30 @@
   z-index: 20;
 }
 
+.skip-to-content {
+  display: block;
+  position: absolute;
+  top: 20px;
+  left: 20px;
+  white-space: nowrap;
+}
+
+.skip-to-content__link {
```

Comment:
For consistency with the rest of the codebase, to visually hide something but keep it accessible, please use the mixin `visuallyhidden` (see https://github.com/elifesciences/pattern-library/blob/develop/assets/sass/_mixins.scss#L140).